docs/webpack-dev-server.config.js

```diff
@@ -51,14 +51,7 @@ const config = {
   module: {
     //eslint loader
     preLoaders: [
-      {
-        test: /\.(js|jsx)$/,
-        loader: 'eslint-loader',
-        include: [path.resolve(__dirname, '../src')],
-        exclude: [
-          path.resolve(__dirname, '../src/svg-icons'),
-        ],
-      },
+
```

Comment:
> Yeah, let's kill this :grin:.

Oops! Must have `git commit -am` after carefully adding the individual files.

If you;re serious about killing this though, I'll leave it. :+1: